### ISSUE TITLE
Error: Cannot find module 'antlr4/atn/index'

### ISSUE THREAD

**Issue by codebubb** [HL7/fhirpath.js#68]
Ran into this problem whilst upgrading an existing Angular application.

It seems a few breaking changes where introduced in antlr4@4.9.0 ([see](https://github.com/antlr/antlr4/issues/2970)) but the problem for the fhirpath package seems to be in relation to the import paths which are overridden [here](https://github.com/HL7/fhirpath.js/blob/master/src/parser/antlr4-index.js).

## Steps to reproduce

With a local project: 

```bash
mkdir fhir-path
cd fhir-path
npm init -y
npm install fhirpath
```

An import: 

```javascript
const fhirpath = require('fhirpath');
```

Then running the script will produce the *Cannot find module* errors.

The same result occurs if using the package is installed globally and called from the terminal.

**Comment by plynchnlm** [comment]
This is now corrected in 2.7.2.  We fixed the antlr version at 4.8 for now.

**Comment by codebubb** [comment]
Thanks very much 👍
